sdk/azurestackhci/Azure.ResourceManager.Hci/api/Azure.ResourceManager.Hci.netstandard2.0.cs

```diff
@@ -68,7 +68,12 @@ public partial class ArcSettingData : Azure.ResourceManager.Models.ResourceData
     {
         public ArcSettingData() { }
         public Azure.ResourceManager.Hci.Models.ArcSettingAggregateState? AggregateState { get { throw null; } }
-        public string ArcInstanceResourceGroup { get { throw null; } }
+        public string ArcApplicationClientId { get { throw null; } set { } }
+        public string ArcApplicationObjectId { get { throw null; } set { } }
+        public string ArcApplicationTenantId { get { throw null; } set { } }
```

Comment:
If we know that the format of these Ids are guid, can you use directive to set them in `autorest.md`. There is existing example:
```
  - from: clusters.json
    where: $.definitions.ClusterProperties.properties
    transform: >
      $.aadClientId.format = "uuid";
      $.aadTenantId.format = "uuid";
      $.cloudId.format = "uuid";
```

Reply:
Fixed